src/chrome/content/rules/yplanapp.com.xml

```diff
@@ -1,11 +1,12 @@
 <ruleset name="YPlan">
 
-	<target host="yplanapp.com" />
-	<target host="www.yplanapp.com" />
+	<target host="*.yplanapp.com" />
 
-  <securecookie host="^(www\.)?yplanapp.com$" name=".+" />
+	<securecookie host="^.*\.yplanapp\.com$" name=".+" />
```

Comment:
No. ([just test it by yourself](https://regex101.com/r/jG4bG2/2))
The issue is just that dot `\.` is always required.
But you can e.g. [fix it like this](https://regex101.com/r/jG4bG2/3).